engine/src/main/resources/swagger/cromwell.yaml

```diff
@@ -219,6 +219,35 @@ paths:
       security:
         - google_oauth:
             - openid
+ '/workflows/${version}/{id}/addLabels':
```

Comment:
Hmm though maybe PATCH is more appropriate:

http://www.restapitutorial.com/lessons/httpmethods.html

(and a million other links)